log/common.go

```diff
@@ -0,0 +1,36 @@
+package log
+
+import (
+	"io"
+	"os"
+)
+
+// For debugging purposes we can change the output-writer
+var stdOut io.Writer
+var stdErr io.Writer
+
+func init() {
+	stdOut = os.Stdout
+	stdErr = os.Stderr
+}
+
+const (
+	lvlWarning = iota - 20
+	lvlError
+	lvlFatal
+	lvlPanic
+	lvlInfo
+	lvlPrint
+)
+
+// Format indicates how the prints will be shown
+var Format = FormatLvl
```

Comment:
mhhh if the golang standard library can give us the function name with line with timestamps and all of that activable, then +1 for that.
-1 for using other external libraries.